services/users/src/routes/company.ts

```diff
@@ -42,8 +61,76 @@ companyRoutes.route("/:id").get(
 companyRoutes.route("/:id").put(
   checkAbility("update", "Company"),
   asyncHandler(async (req, res) => {
-    const updatedCompany = await CompanyModel.findByIdAndUpdate(req.params.id, req.body, {
-      new: true,
+    const requestedCompany = await CompanyModel.findById(req.params.id);
+
+    if (!requestedCompany) {
+      throw new BadRequestError("Requested company not valid");
+    }
+
+    const userCompany = await apiCall(
+      Service.USERS,
+      { method: "GET", url: `/companies/employees/${req.user?.uid}` },
+      req
+    );
```

Comment:
You're already in the users service, so you don't need to make this api call, you can just access the database directly and get the info u need. If you need to keep writing the same code in multiple endpoints, you can just make this into a separate function.